client/src/api/unculledZonesSlice.ts

```diff
@@ -0,0 +1,36 @@
+import { RootState } from "../app/store";
+import { gameLoaded, gameUnloaded } from "./actions";
+import { UnculledZone } from "./liberationApi";
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+
+interface UnculledZonesState {
+  zones: UnculledZone[];
+}
+
+const initialState: UnculledZonesState = {
+  zones: [],
+};
+
+export const unculledZonesSlice = createSlice({
+  name: "unculledZonesState",
+  initialState,
+  reducers: {
+    updated: (state, action: PayloadAction<UnculledZone[]>) => {
+      state.zones = action.payload;
+    },
+  },
+  extraReducers: (builder) => {
+    builder.addCase(gameLoaded, (state, action) => {
+      state.zones = action.payload.unculled_zones;
+    });
+    builder.addCase(gameUnloaded, (state) => {
+      state.zones = initialState.zones;
+    });
```

Comment:
Updates for when the game is loaded/unloaded, this is why the API of Game had to include the `unculled_zones`